src/ansys/tools/report/report.py

```diff
@@ -80,15 +80,15 @@ def __init__(
         """Report class constructor."""
         # Mandatory packages
         core = [
+            "appdirs",
+            "google.protobuf",  # protobuf library
+            "grpc",  # grpcio
             "matplotlib",
             "numpy",
-            "pyvista",
-            "appdirs",
-            "tqdm",
             "pyiges",
+            "pyvista",
             "scipy",
-            "grpc",  # grpcio
-            "google.protobuf",  # protobuf library
+            "tqdm",
```

Comment:
Why the order matters??

Reply:
Alphabetical order is always better than no order 😄

Reply:
Loooll... true!